charts/opentelemetry-kube-stack/values.yaml

```diff
@@ -307,8 +307,12 @@ defaultCRConfig:
   #   metrics:
   #     enableMetrics: true
 
-  # Update strategy for the collector
-  updateStrategy: {}
```

Comment:
@jaronoff97 this is a breaking change, are you ok with that? In the collector chart we'd handle this by supporting both for a bit. Do you want to do that here?

Reply:
@TylerHelmuth @byehn, I do think we should have a migration path here actually. Let's add in the two new strategies, and put a note in the NOTES to inform users of the deprecation. Sorry for the delayed response.